bin/MadGraph5_aMCatNLO/cards/WW_helicity/WW_LL/WW_LL_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+define l- = mu- e- ta-
+define l+ = mu+ e+ ta+
+
+generate p p > w+{0} w-{0} $$ h
+
+output WW_LL
```

Comment:
the default model is 4 flavor, why not use 5f here.

Reply:
Hi Meng, This might have issues with tW contamination. No?

Reply:
thanks Sihyun, yes, it's a LO one.